### PR TITLE
qt5-build.eclass: Support EAPI-8

### PR DESCRIPTION
**Pull request by a17r**
I don't see anything else to change.

### CODE REVIEW

**Review by Pesa — COMMENTED** [review]
`RESTRICT+=...` is no longer needed in EAPI 8 because the variable is now accumulated automatically... but I suppose it doesn't hurt either?

**Comment by a17r** [comment]
We could change that with dropping EAPI-7.

**Comment by a17r** [comment]
> `RESTRICT+=...` is no longer needed in EAPI 8 because the variable is now accumulated automatically... but I suppose it doesn't hurt either?

Appended EAPI-8 bumps to dev-qt/* and dropped EAPI-7 support in subsequent commit.

File: dev-qt/qtdeclarative/qtdeclarative-5.15.9999.ebuild [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # Copyright 2009-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=7
+EAPI=8
 
-PYTHON_COMPAT=( python3_{6..9} )
+PYTHON_COMPAT=( python3_{7..10} )
```
Comment (Pesa) [inline dev-qt/qtdeclarative/qtdeclarative-5.15.9999.ebuild:6]: 3.7 is gone so you can start from 8

File: dev-qt/qtnetwork/qtnetwork-5.15.9999.ebuild [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # Copyright 1999-2021 Gentoo Authors
```
Comment (Pesa) [inline dev-qt/qtnetwork/qtnetwork-5.15.9999.ebuild:1]: (stupid github doesn't let me add comments on lines outside the immediate context of the diff...)

Around lines 55-60 there are 3 cases of `usex` that can be turned into `usev` dropping the third (empty) argument.

File: dev-qt/qtsql/qtsql-5.15.9999.ebuild [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # Copyright 1999-2021 Gentoo Authors
```
Comment (Pesa) [inline dev-qt/qtsql/qtsql-5.15.9999.ebuild:1]: another case of `usex` -> `usev` below

File: dev-qt/qtwidgets/qtwidgets-5.15.9999.ebuild [hunk 4]
```diff
@@ -1,7 +1,7 @@
 # Copyright 1999-2021 Gentoo Authors
```
Comment (Pesa) [inline dev-qt/qtwidgets/qtwidgets-5.15.9999.ebuild:1]: another `usex` -> `usev` on line 54

**Review by Pesa — APPROVED** [review]
very minor comments, no need to amend these commits, you can push follow-up patches

and thanks a lot!

**Comment by Pesa** [comment]
btw your signature shows up as "unverified" on github

**Pull request merged**